app/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-permission android:name="android.permission.INTERNET"/>
+
     <application
+        android:name=".RecipeApps"
         android:allowBackup="true"
```

Comment:
pls allowBackup false if it not need to backup , for more infos
https://medium.com/@jecky999/understanding-the-allowbackup-property-in-android-manifest-958151819ea

Reply:
setting the allow back up to false as we don't want to integrate the back up 9baf80468f57d78d6c4b13f545e1ebaa10f9d59c